doc/assets/css/common/fonts.scss

```diff
@@ -1,18 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Lato:400,700&display=swap');
-
-@font-face {
-  font-family: 'LatoFallback';
-  src: url(../fonts/Lato-Regular.woff2) format('woff2'), url(../fonts/Lato-Regular.woff) format('woff'), url(../fonts/Lato-Regular.ttf) format('ttf');
-  font-weight: normal;
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'LatoFallback';
-  src: url(../fonts/Lato-Bold.woff2) format('woff2'), url(../fonts/Lato-Bold.woff) format('woff'), url(../fonts/Lato-Bold.ttf) format('ttf');
-  font-weight: bold;
-  font-style: normal;
-}
+@import url('https://fonts.googleapis.com/css?family=Inter:400,700&display=swap');
```

Comment:
Please use add the font as an asset and then load it locally. We want to avoid unnecessary requests to google if we can.